docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -154,6 +154,117 @@ A successful `curl` return
 
 The TypeScript SDK provides a wrapper for the `suix_queryEvents` method: [`client.queryEvents`](https://sdk.mystenlabs.com/typedoc/classes/_mysten_sui.client.SuiClient.html#queryEvents).
 
+<details>
+<summary>
+    TypeScript SDK queryEvents example
+</summary>
+
+```
+import {
+    useSignAndExecuteTransactionBlock,
+    useSuiClient,
+} from "@mysten/dapp-kit";
+import { TransactionBlock } from "@mysten/sui.js/transactions";
+import { Button, Flex, Heading } from "@radix-ui/themes";
+
+export function Creategame({
+    onCreated,
+}:{
+    onCreated: (id: string) => void;
+}) {
+    const { mutate: signAndExecute } = useSignAndExecuteTransactionBlock();
+    const client = useSuiClient();
+
+    const executeMoveCall = (method: "small" | "large") => {
+        const txb = new TransactionBlock();
+
+        if (method === "small") {
+            txb.moveCall({
+                arguments: [txb.pure.u64(0)],
+                target: `<PACKAGE-ID>::<MODULE>::create_game`,
+            });
+        } else {
+            txb.moveCall({
+                arguments: [txb.pure.u64(1)],
+                target: `<PACKAGE-ID>::<MODULE>::create_game`,
+            });
+        }
+
+        signAndExecute(
+            {
+                transactionBlock: txb,
+                options: {
+                    showEffects: true,
+                    showObjectChanges: true,
+                },
+            },
+            {
+                onSuccess: (tx) => {
+                    client.
+                    queryEvents({
+                        query:{Transaction:tx.digest}
+                    }).then(PaginatedEvents=> {
+                        const events = JSON.stringify(PaginatedEvents.data[0].parsedJson);
+
+                        if (events.length > 0) {
+                            onCreated(events)
+                        } else {
+                            onCreated('No events found for the given criteria.');
+                        }
+                    })
+                        .catch(error => {
+                            console.error('Error querying events:', error);
+                        });
+                },
+            },
+        );
+    };
+    return (
+        <>
+            <Heading size="3">Game Start</Heading>
+
+            <Flex direction="column" gap="2">
+                <Flex direction="row" gap="2">
+                    <Button onClick={() => executeMoveCall("small")}>
+                        small
+                    </Button>
+                    <Button onClick={() => executeMoveCall("large")}>
+                        large
+                    </Button>
+                </Flex>
+            </Flex>
+        </>
+    );
+}
+```
+</details>
+
+<details>
+<summary>
+    TypeScript SDK queryEvents Response
```

Comment:
The extra click for the reader is not necessary here. Suggest adding this content (minus that last line) to the bottom of the `details` above.

Reply:
Thanks @ronny-mysten , I appreciate your feedback on my issue. I will be more mindful of it in the future.